cloud-formation/cfn.yaml

```diff
@@ -127,6 +127,7 @@ Resources:
                 - states:ListStateMachines
                 - states:StartExecution
                 - states:GetExecutionHistory
+                - states:DescribeStateMachine
                 Resource: arn:aws:states:*:*:*
```

Comment:
I know you haven't changed this here, but should we be more restrictive about which Step Functions this app can trigger?

Reply:
That would be nice, although that will then need to be stored in the app config and in the cloudformation